source/elements/oneDPL/source/parallel_api.rst

```diff
@@ -262,6 +262,19 @@ Iterators
 The oneDPL iterators are defined in the :code:`<oneapi/dpl/iterator>` header,
 in :code:`namespace oneapi::dpl`.
 
+Let us define a named requirement, :code:`ValidParallelIteratorSource`, to describe valid random access iterator-like
+types that can be used as source for oneDPL's iterators as described below.
+The type :code:`Iter` satisfies the :code:`ValidParallelIteratorSource` named requirement if it satisfies
+at least one of the following:
```

Comment:
Change "at least one" to "any"? Perhaps not much difference, but "at least one" confuses me a bit because the items in the list are seemingly exclusive.

Reply:
"any" is fine.  I think that `counting_iterator` also satisfies "random access iterator" for instance. The overlap is generally only between "random access iterator" and other types.